packages/prettier-plugin-java/scripts/update-test-output.js

```diff
@@ -5,7 +5,10 @@ const path = require("path");
 const fs = require("fs");
 const prettier = require("prettier");
 
-const samplesDir = path.resolve(__dirname, "../test");
+const samplesDir =
+  process.argv.indexOf("-single") > -1
+    ? path.resolve(__dirname, "./single-printer-run")
```

Comment:
should we exclude ./single-printer-run folder from the source control using .gitignore